MVP_Clicker/Assets/Project/Scripts/Game/Areas/Skill/View/SkillView.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Project.Scripts.Game.Areas.Skill.View
+{
+    public class SkillView : MonoBehaviour, ISkillView
+    {
+        public event Action Activated;
+
+        private float _currentShadowIntensity;
+
+        [SerializeField] private Image _cooldownShadow;
+        [SerializeField] private TextMeshProUGUI _description;
+        [SerializeField] private TextMeshProUGUI _name;
+        [SerializeField] private TextMeshProUGUI _recoveryDuration;
+        [SerializeField] private Image _skillIcon;
+
+        public string Description
+        {
+            get => _description.text;
+            set => _description.text = value;
+        }
+
+        public string Name
+        {
+            get => _name.text;
+            set => _name.text = value;
+        }
+
+        public int RecoveryDuration
+        {
+            set => _recoveryDuration.text = value.ToString();
+        }
+
+        public Sprite SkillIcon
+        {
+            get => _skillIcon.sprite;
+            set => _skillIcon.sprite = value;
+        }
+
+        public void ActivateSkill()
```

Comment:
Maybe more good to subscribe to button via serizlied field?
It's looks strange that we have public method ActivateSkill and I can't immediatly understand that someone from unity call it